src/main/resources/Documentation/about.md

```diff
@@ -1,41 +1,55 @@
-TODO: Add Plugin introduction
+Eiffel Gerrit plugin is may be used to send Eiffel events when changes occurs in in a project in Gerrit.
+This plugin simplifies communication in continuous integration and delivery pipelines and helps keeping
+source code management traceability.
 
 For more Eiffel information follow this link:
 <https://github.com/eiffel-community/eiffel-gerrit-plugin>
 
+#####Eiffel Events:
+
+Version 0.0.1
```

Comment:
We might want to consider linking to git tags/releases in GitHub later on when they exist.

Reply:
Yes, and most likely this will render a 1.0.0 release and not 0.0.1.